sdk/documenttranslation/azure-ai-documenttranslation/tests/testcase.py

```diff
@@ -19,4 +50,70 @@ class DocumentTranslationTest(AzureTestCase):
     def __init__(self, method_name):
         super(DocumentTranslationTest, self).__init__(method_name)
         self.vcr.match_on = ["path", "method", "query"]
+        self.recording_processors.append(OperationLocationReplacer())
+        self.storage_name = os.getenv("DOCUMENTTRANSLATION_STORAGE_NAME", "redacted")
+        self.storage_endpoint = "https://" + self.storage_name + ".blob.core.windows.net/"
+        self.storage_key = os.getenv("DOCUMENTTRANSLATION_STORAGE_KEY")
+        self.scrubber.register_name_pair(
+            self.storage_endpoint, "https://redacted.blob.core.windows.net/"
+        )
+        self.scrubber.register_name_pair(
+            self.storage_name, "redacted"
+        )
+        self.scrubber.register_name_pair(
+            self.storage_key, "fakeZmFrZV9hY29jdW50X2tleQ=="
+        )
```

Comment:
If these tests use preparers, these are unnecessary. The preparer does this for you [here](https://github.com/Azure/azure-sdk-for-python/blob/master/tools/azure-sdk-tools/devtools_testutils/powershell_preparer.py#L74-L76)

Reply:
This was paranoia on my part because the storage endpoint was not getting scrubbed in the recording. Now that I'm building the storage endpoint in the code (since ARM template returned it with weird concatenation), I think I do need to have it scrubbed here?

Reply:
Yes you are correct, I read this too quickly. This approach works, you can add this to your preparer if you want.